test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -617,8 +617,11 @@ export class DashboardPageControls extends FtrService {
   }
 
   // Time slider functions
-  public async gotoNextTimeSlice() {
+  public async gotoNextTimeSlice(closePopover: boolean = true) {
```

Comment:
I would not recommend adding closePopover parameter to gotoNextTimeSlice. It takes away from the readability of the functional tests. For example
* gotoNextTimeSlice(false) means goto next time slice
* gotoNextTimeSlice() means goto next time slice and close popover.

When you come back to look at these tests in a few months, you will not remember what passing false and not passing false actually do.
I think this does not really save a lot and showing explicit steps in functional tests really helps with readability.

Reply:
The reason I added this is because there was some flakiness around opening and closing popovers in rapid succession, so giving the programmer control over when to close the popover and when to keep it open has improved this for me in the past. This is also a pattern in `dashboard_page_controls` - for example, `shouldSearch` in `controlsEditorSetfield` and `filterOutExists` in `optionsListPopoverGetAvailableOptions`

I totally get what you're saying about readability, though. As part of this PR, I could instead make it accept an `options` object, so that the use is more explicit? So the call would look something like `gotoNextTimeSlice({ closePopover: true })` instead, and I could modify the other two functions I mentioned to follow this pattern.